src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -357,15 +357,22 @@ def post(self, emulator=None):
 echo "Retrieving SCRAM project..."
 # do the actual executing
 $SCRAM_COMMAND project $SCRAM_PROJECT $CMSSW_VERSION
-if [ $? -ne 0 ]; then echo "Scram failed"; exit 71; fi
+EXIT_STATUS=$?
+if [ $EXIT_STATUS -ne 0 ]; then echo "Scram failed with exit code: $EXIT_STATUS"; exit 71; fi
```

Comment:
Hmm, interesting :) I don't like repeating this `EXIT_STATUS=$?` line 1 million times, but at least it's very clear.